apps/server/server.cpp

```diff
@@ -33,12 +33,16 @@
 #include "aditof/aditof.h"
 #include "aditof/sensor_enumerator_factory.h"
 #include "aditof/sensor_enumerator_interface.h"
-#include "buffer.pb.h"
 
 #include "../../sdk/src/connections/utils/connection_validator.h"
-
+#include "buffer.pb.h"
 #include <csignal>
+
+#ifndef JS_BINDINGS
```

Comment:
Server builds only on target. Javacript bindings are disabled on target so this check doesn't make sense.